README.md

```diff
@@ -27,8 +27,8 @@ Blood port based on EDuke32
 3. Optionally, if you want to use CD audio tracks instead of MIDI, provide FLAC/OGG recordings in following format: bloodXX.flac/ogg, where XX is track number. Make sure to enable Redbook audio option in sound menu.
 4. Optionally, if you want cutscenes and you have the original CD, copy the `movie` folder into NBlood's folder (the folder itself too).
 If you have the GOG version of the game, do the following:
-   * make a copy of `game.ins` (or `game.inst`) named `game.cue`
-   * mount the `.cue` as a virtual CD (for example with `WinCDEmu`)
+   * make a copy of `game.ins` (or `game.inst`) named `game.bin`
+   * mount the `.bin` as a virtual CD (for example with `WinCDEmu`)

```

Comment:
This change is not correct. The `game.ins` (GOG version) or `game.inst` (Steam OUWB) corresponds to the .cue file. The .bin file is named `game.gog` (in both versions...) and does not need any renaming since the cue file refers to it by that exact name.